uktrade/data-hub-frontend
# Reactify OMIS billing address form

## Description of change

The form for editing the billing address for an OMIS order has been reactified. The VAT status confirmation page has also been reactified as it was only accessible from the billing address page.

## Test instructions

Open an OMIS order and go to the invoice details edit page, then edit the billing address (without changing the country). It should redirect to the invoice details edit page and show the new address.

Edit the billing address again and change the country. It should redirect you to the VAT confirmation page.

## Screenshots

<details>
<summary>Billing address</summary>

### Before

<img width="753" alt="Screenshot 2023-10-24 at 13 08 24" src="https://github.com/uktrade/data-hub-frontend/assets/36161814/fbabd51c-5593-4ac2-a86d-b28212da4a7d">


### After

![Screenshot 2023-10-24 at 13-08-00 Edit billing address - JTJ148_23 - Orders (OMIS) - DBT Data Hub](https://github.com/uktrade/data-hub-frontend/assets/36161814/f30f8cbf-b453-4b49-8aea-7291c2af72b9)

</details>

<details>
<summary>VAT confirmation</summary>

### Before

<img width="742" alt="Screenshot 2023-10-24 at 13 07 05" src="https://github.com/uktrade/data-hub-frontend/assets/36161814/bbb79653-91b3-4111-a17b-f4dde0621273">

### After

<img width="988" alt="Screenshot 2023-10-24 at 13 09 07" src="https://github.com/uktrade/data-hub-frontend/assets/36161814/cf3b8e66-ffea-4b8e-b7fc-2670d8743320">

</details>


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
